doc/verilog_systemverilog.txt

```diff
@@ -448,6 +448,19 @@ Example:
     Note: The commands |:VerilogFoldingAdd| and |:VerilogFoldingRemove| are
     provided to allow an easier management of this variable.
 
+                                                      *g:verilog_quick_syntax*
+g:verilog_quick_syntax~
+Default: undefined
+
+    Whena enabled, syntax regions will not be defined with Vim's 'syntax'
+    command. This can help performance if you are using an old machine or
+    veiwing a large file such as a netlist. This can be useful if you don't
+    care about automatic indentation but still wish to have syntax coloring.
```

Comment:
I see your point and agree with it.

The only thing that scares me is that I already have some colleagues complaining about Vim's syntax highlight speed. Hope this doesn't make everything even worse :(